platforms/web/lib/useListeners/event.ts

```diff
@@ -146,7 +96,12 @@ export function handleInput(
     testUtilities: TestUtilities,
     formattingFunctions: FormattingFunctions,
     inputEventProcessor?: InputEventProcessor,
-) {
+):
+    | {
+          content: string | undefined;
```

Comment:
`content?: string`

Reply:
Done, and also I added a rename, which I'd be grateful if you could review too please @florianduros